osquery/tables/system/posix/apt_sources.cpp

```diff
@@ -19,51 +19,95 @@
 namespace osquery {
 namespace tables {
 
-Status parseAptSourceLine(const std::string& line, AptSource& apt_source) {
-  auto comp = osquery::split(line, " ");
-  if (comp.empty() || comp[0][0] == '#') {
-    return Status::failure("Cannot parse comment");
+Status parseAptSourceLine(const std::string& input_line,
```

Comment:
Since we're not using `input_line` for anything, it can be passed as a mutable non-const. Then you can avoid using `std::string line` below and just overwrite this value.

Reply:
i knew there was a reason i did this.  The reason I wanted to leave the input var as const and copy it, is that the tests pass in const char strings See https://github.com/osquery/osquery/pull/6482/files#diff-7f48f8efdffa1486a92073f2a776148aR105  The strings are small, so I expect that the extra overhead of making a copy isn't big.